### ISSUE TITLE
Wrong modelview when using pip version

### ISSUE THREAD

**Issue by d4nst** [patrikhuber/eos#219]
I am using eos 3D model to generate 2D depth maps. Initially, I used the python bindings compiled from source and everything worked fine. However, when using the version from pip, there is a problem with the modelview matrix that makes my code not work properly. Below you can see the difference between executing ``demo.py`` using the bindings compiled from source and the ones from pip (both of them are version 0.16.1):

- Source:
```
ipdb> pose.get_modelview()
array([
[  0.87695104,   0.04230939,  -0.47871366, 309.55637   ],      
[ -0.09763256,   0.9910292 ,  -0.0912636 , 312.72763   ],       
[  0.4705579 ,   0.12677175,   0.87321484,   0.        ],      
[  0.        ,   0.        ,   0.        ,   1.        ]],  dtype=float32)
```
- Pip:
```
ipdb> pose.get_modelview()
array([
[  0.     ,   0.     ,   0.     , 309.55637], 
[  0.     ,   0.     ,   0.     , 312.72763],       
[  0.     ,   0.     ,   0.     ,   0.     ],       
[  0.     ,   0.     ,   0.     ,   0.     ]], dtype=float32)
```

**Comment by patrikhuber** [comment]
Hi, thanks for the report!

I think this needs further investigation. I don't think I currently have time to investigate.
Can you just use the source-compiled version in the meantime?

**Comment by patrikhuber** [comment]
I've had one more person report issues with the projection being wrong in `Release` builds of the Python bindings. I am marking this as potential bug and will try to find the time to debug.

**Comment by rstar000** [comment]
Hi! Maybe i'm a bit late, but it seems like the model-view matrix is actually inverted. This is the result of projecting model axes using the returned model-view matrix, but inverted. 
![image](https://user-images.githubusercontent.com/20645210/58951382-a5a04180-8799-11e9-80d6-0323b581f9f1.png)

Code for reference: 
`def project_axis(matrix):   
    axis = np.float32([[500,0,0], 
                          [0,500,0], 
                          [0,0,500]])
               
    proj_points = matrix[:3,:3].T @ axis
    return proj_points[:3,:2]
`

**Comment by patrikhuber** [comment]
Hi @rstar000,

I think this is a different (potential) issue from the one that was originally reported here.

With "inverted", do you mean transposed? I would be quite surprised if the model-view matrix was transposed. Is it possible that you got fooled by the depiction of the three coloured drawn axes? They can often look like they point in the "wrong" direction, our brain's interpretation can fool us when only looking at a static image. I would suggest that you run it on a video where someone turns their head (or e.g. the live cam), it's much easier to see then whether the axes are drawn correctly.

If you still think there is an issue, could you open a separate issue for it to continue the discussion there?

Thanks!

**Comment by rstar000** [comment]
@patrikhuber  Yes, I mean the rotation matrix is inverted (or transposed, they would be the same in case of orthonormal matrices). That's strange. I don't think i'm interpreting the result incorrectly. I looked at many examples across very different poses. The code without transposing the matrix looks completely off. 

Do you think my logic might be off? I want to project the axes from model space to screen space. For that I use the model-view matrix to do orthographic projection. Seems reasonable?

Here are some examples when projecting without transposition:
![image](https://user-images.githubusercontent.com/20645210/58952676-0aa96680-879d-11e9-92de-9851442119fc.png)
![image](https://user-images.githubusercontent.com/20645210/58952738-33316080-879d-11e9-9d45-e56779f8a2a8.png)
![image](https://user-images.githubusercontent.com/20645210/58952782-59570080-879d-11e9-874d-d4f22f8d199b.png)

**Comment by rstar000** [comment]
@patrikhuber Hey, sorry, I just found a silly bug in my projection code. I was using row vectors instead of column vectors for some reason. The matrix returned from pose.get_modelview() is absolutely correct!

**Comment by patrikhuber** [comment]
@rstar000 Great! :-)
